docs/source/modules/authentication.rst

```diff
@@ -22,6 +22,34 @@ If you want to use overrided rest_registration views, replace rest_registration
     ...
   ]
 
+Custom rest api with session
+----------------------------
+
+There is overridden SessionMiddleware available, to enable REST API with the session.
```

Comment:
We support two methods of maintaining session information for your client: cookie-based and header-based.

When you perform the account/login function, you can choose whether the function should return a session cookie or whether session id should be returned in response JSON.

If you choose the cookie, you will then need to supply the cookie to all subsequent requests. Likewise, if you opt for session id as a variable, you will have to provide Authorization header to all subsequent requests.

Our modified SessionMiddleware only overrides Django's as much to also accept the Authorization header and clears the session and CSRF cookies